schedule/sles4sap/netweaver/netweaver_cluster_node.yaml

```diff
@@ -38,6 +38,7 @@ schedule:
   - console/consoletest_setup
   - console/check_os_release
   - console/hostname
+  - sles4sap/selinux_set_permissive
```

Comment:
Will you remove this before merging?

Reply:
Thanks for your review!
Yes, I will update MR to remove that.